### ISSUE TITLE
Multiple containers & inheritance

### ISSUE THREAD

**Issue by onionhammer** [AzureCosmosDB/data-migration-desktop-tool#90]
Is it possible to inherit from a root setting in your JSON configuration so that you can list out multiple conatiner source/sinks without repeating connection information?

**Comment by bowencode** [comment]
Yes, the Operations array in configuration can be used for multiple sources or sinks in the same job and those will each inherit from the root SourceSettings and SinkSettings. You can see some specific examples here: https://codemindinterface.com/2023/03/cosmos-tool-operations/
